mail/header.go

```diff
@@ -308,6 +364,35 @@ func (h *Header) MsgIDList(key string) ([]string, error) {
 	return l, nil
 }
 
+// MsgIDList parses a list of URLs from a list command header. It returns URLs.
+// If the header field is missing, it returns nil.
+//
+// This can be used on List-Help, List-Unsubscribe, List-Subscribe, List-Post,
+// List-Owner, and List-Archive headers.
+//
+// See https://www.rfc-editor.org/rfc/rfc2369 for more information.
+//
+// In the case that the value of List-Post is the special value, "NO", the
+// return value is a slice containing one element, nil.
+func (h *Header) ListCommandURLList(key string) ([]*url.URL, error) {
```

Comment:
I wonder if we should just use the name `ListCommand`, and leave out the `URLList` part.